helix-view/src/theme.rs

```diff
@@ -52,21 +52,24 @@ impl Loader {
     }
 
     /// Loads a theme searching directories in priority order.
-    pub fn load(&self, name: &str) -> Result<Theme> {
+    pub fn load(&self, name: &str) -> Result<(Theme, Vec<String>)> {
```

Comment:
Since we have many callers that need to handle the warnings let's create a `load_with_warnings` helper that has this function's body and then `load` can call that helper and log the warnings.